modules/common/css/lib/errors.scss

```diff
@@ -259,7 +259,7 @@
     margin: -4px -12px;
     padding: 4px 12px;
     text-align: left;
-    margin-top: -12px;
+    margin-top: -4px;
```

Comment:
Nit: We already set the top margin to `-4px` above. Can we just remove this line?